core/src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -245,7 +251,18 @@ $(function () {
         type: 'POST',
         url: serviceInfo.debugPath,
         data: request,
-        headers: httpHeaders,
+        beforeSend: function (xhr) {
+          for (var name in httpHeaders) {
+            if (httpHeaders.hasOwnProperty(name)) {
+              var values = httpHeaders[name];
+              if (Array.isArray(values)) {
+                xhr.setRequestHeader(name, values.join());
```

Comment:
Thanks for verifying it doesn't work, even though the spec says it should...

https://fetch.spec.whatwg.org/#concept-header-list-combine
https://xhr.spec.whatwg.org/#the-setrequestheader()-method

Anyways, I learned that multiple headers are just a comma-separated list - I thought they'd be multiple header lines with the same key :)